### ISSUE TITLE
Wrong path of proto file

### ISSUE THREAD

**Issue by nigeldelviero** [merkle3/merkle-sdk-ts#2]
```
Error: ENOENT: no such file or directory, open '../proto/broker.proto'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at fetch (C:\Users\delvi\Projects\flash-v1\node_modules\protobufjs\src\root.js:173:34)
    at Root.load (C:\Users\delvi\Projects\flash-v1\node_modules\protobufjs\src\root.js:207:13)
    at Root.loadSync (C:\Users\delvi\Projects\flash-v1\node_modules\protobufjs\src\root.js:248:17)
    at loadProtosWithOptionsSync (C:\Users\delvi\Projects\flash-v1\node_modules\@grpc\proto-loader\src\util.ts:79:27) 
    at loadSync (C:\Users\delvi\Projects\flash-v1\node_modules\@grpc\proto-loader\src\index.ts:391:47)
    at Transactions.stream (C:\Users\delvi\Projects\flash-v1\node_modules\@mkl3\sdk\dist\cjs\transactions.js:18:63)   
    at C:\Users\delvi\Projects\flash-v1\src\index.ts:93:6
    at Generator.next (<anonymous>) {
  errno: -4058,
  syscall: 'open',
  code: 'ENOENT',
  path: '../proto/broker.proto'
}
```

Source: @mkl3\sdk\dist\cjs\transactions.js:18:63

**Comment by nigeldelviero** [comment]
For now it will works if I put the proto directory to my project directory.

**Comment by Theo-** [comment]
Fixed in version `1.0.11` of the SDK. Thanks for the report!
